util/setup_base.txt

```diff
@@ -40,60 +42,49 @@ const config = {
     { level: 2,
       // This is the name of the role.
-      name: "Moderator",
+      name: 'Trainee',
       // The following lines check the guild the message came from for the roles.
       // Then it checks if the member that authored the message has the role.
       // If they do return true, which will allow them to execute the command in question.
       // If they don't then return false, which will prevent them from executing the command.
       check: (message) => {
         try {
-          const modRole = message.guild.roles.find(r => r.name.toLowerCase() === message.settings.modRole.toLowerCase());
-          if (modRole && message.member.roles.has(modRole.id)) return true;
+          const traineeRole = message.guild.roles.find(r => r.name.toLowerCase() === message.settings.traineeRole.toLowerCase());
+          if (traineeRole && message.member.roles.has(traineeRole.id)) return true;
         } catch (e) {
           return false;
         }
       }
     },
 
     { level: 3,
-      name: "Administrator", 
+      name: 'Council', 
       check: (message) => {
         try {
-          const adminRole = message.guild.roles.find(r => r.name.toLowerCase() === message.settings.adminRole.toLowerCase());
-          return (adminRole && message.member.roles.has(adminRole.id));
+          const councilRole = message.guild.roles.find(r => r.name.toLowerCase() === message.settings.councilRole.toLowerCase());
+          return (councilRole && message.member.roles.has(councilRole.id));
         } catch (e) {
           return false;
         }
       }
     },
-    // This is the server owner.
-    { level: 4,
-      name: "Server Owner", 
-      // Simple check, if the guild owner id matches the message author's ID, then it will return true.
-      // Otherwise it will return false.
-      check: (message) => message.channel.type === "text" ? (message.guild.owner.user.id === message.author.id ? true : false) : false
-    },
-
-    // Bot Support is a special inbetween level that has the equivalent of server owner access
-    // to any server they joins, in order to help troubleshoot the bot on behalf of owners.
-    { level: 8,
-      name: "Bot Support",
-      // The check is by reading if an ID is part of this array. Yes, this means you need to
-      // change this and reboot the bot to add a support user. Make it better yourself!
-      check: (message) => config.support.includes(message.author.id)
-    },
-
     // Bot Admin has some limited access like rebooting the bot or reloading commands.
     { level: 9,
-      name: "Bot Admin",
-      check: (message) => config.admins.includes(message.author.id)
+      name: 'Police', 
+      check: (message) => {
+        try {
+          const policeRole = message.guild.roles.find(r => r.name.toLowerCase() === message.settings.policeRole.toLowerCase());
+          return (policeRole && message.member.roles.has(policeRole.id));
```

Comment:
If you don't feel like going through with Ian's suggestion, this entire block can be replaced with:

```js
message.member.roles
  .find(role => role.name.toLowerCase(), message.settings.policeRole.toLowerCase())
```